infra/azure/terraform/backend.tf

```diff
@@ -34,8 +34,8 @@ resource "azurerm_container_group" "titus-backend-containergroup" {
       "PG_DB"       = azurerm_postgresql_database.titus-db.name
       "PG_USER"     = "${var.db_user}@${azurerm_postgresql_server.titus-db-server.name}"
       "PG_PASS"     = azurerm_key_vault_secret.titus-db-password.value
```

Comment:
just as a sanity check, this variable is used for what?

Reply:
removed, it is used only when `SECRETS_STRAGEGY == env`

Reply:
we also have one similarly named in db-manager, can you check if that one is used as well?

Reply:
checked and not used

Reply:
shall we remove it then?